devices/tuya.js

```diff
@@ -4151,4 +4151,14 @@ module.exports = [
                 .withDescription('Volume of the alarm'),
         ],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0041', ['_TZ3000_fa9mlvja']),
```

Comment:
This seems to be the same as https://github.com/Koenkk/zigbee-herdsman-converters/blob/2998f7b1c91239594b9d76710643249e2429bd28/devices/tuya.js#L1763

Reply:
but without 'hold' action

Reply:
Ah I see, thanks!